### PR TITLE
Minor fixes to get setup.py to work on Windows

### PR DESCRIPTION
**Pull request by dhirschfeld**
I'm attempting to get JupyterHub up & running on Windows. This was the first hurdle I've come across...

### CODE REVIEW

**Comment by dhirschfeld** [comment]
Fix shamelessly stolen from jupyter-widgets/declarativewidgets#380

**Comment by dhirschfeld** [comment]
I think the CI error is unrelated?

File: setup.py [hunk 1]
```diff
@@ -179,13 +180,11 @@ def run(self):
         
         env = os.environ.copy()
         env['PATH'] = npm_path
-        
+        args = ['bower', 'install', '--allow-root', '--config.interactive=false']
+        if os.name == 'nt':
+            args = ['cmd', '/C'] + args
```
Comment (minrk) [inline setup.py]: I think `shell=True` should do this, right?

File: setup.py [hunk 2]
```diff
@@ -119,7 +119,8 @@ def get_package_data():
 from distutils.command.sdist import sdist
 
 
-npm_path = ':'.join([
+sep = ';' if os.name == 'nt' else ':'
```
Comment (minrk) [inline setup.py]: This should be `os.pathsep`, I think

File: setup.py (hunk 1 above)
Comment (dhirschfeld) [inline setup.py]: That sounds plausible! I'll give it a go and report back...

File: setup.py (hunk 2 above)
Comment (dhirschfeld) [inline setup.py]: Yep. I incorrectly thought it was for forward/back slashes. Will change...

File: setup.py (hunk 1 above)
Comment (dhirschfeld) [inline setup.py]: That worked and is cleaner. New commit pushed...

**Comment by dhirschfeld** [comment]
CI failure seems unrelated `MemoryError`

**Pull request merged**